tests/postcards_test.js

```diff
@@ -95,7 +95,7 @@ test("create, list, read then cancel a postcard", async function (t) {
   await deleteAddress(from);
 });
 
-test("creates a postcard given a local filepath as the front", async function (t) {
+test.only("creates a postcard given a local filepath as the front", async function (t) {
```

Comment:
definitely don't want to leave this in place!

Reply:
OOPS, getting rid of it now!